tests/acceptance/stepDefinitions/sharingContext.js

```diff
@@ -793,8 +793,8 @@ Then(
       expectedStatus === 'Accepted'
         ? SHARE_STATE.accepted
         : expectedStatus === 'Declined'
-        ? SHARE_STATE.declined
-        : SHARE_STATE.pending
+          ? SHARE_STATE.declined
+          : SHARE_STATE.pending
```

Comment:
I actually really like this change.

Makes cascading ternary ifs (which I really dislike) a bit less unreadable..